docs/fsharp/language-reference/compiler-messages/fs0003.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Compiler Error FS0003"
+ms.date: 12/23/2019
+f1_keywords:
+  - "FS0003"
+helpviewer_keywords:
+  - "FS0003"
+---
+
+# FS0003: This value is not a function and cannot be applied
+
+This message is given when a parameter is given to a value that cannot accept parameters.  This is often the case when attempting to use an indexer into a collection like a list, array, or IEnumerable.
+
+The following code demonstrates one way the error can be seen:
+
+[!code-fsharp[FS0003-pass-parameter-to-value](~/samples/snippets/fsharp/compiler-messages/fs0003.fs#2-4)]
+
+In the previous snippet, the FS0003 error is shown because the value `v` is not a function that accepts parameters.
+
+```text
+error FS0003: This value is not a function and cannot be applied.
+```
+
+Another way the error can be seen is when incorrectly using the indexer available on a type:
+
+[!code-fsharp[FS0003-indexer-missing-dot](~/samples/snippets/fsharp/compiler-messages/fs0003.fs#7-8)]
+
+In the previous snippet the FS0003 error is shown because in F# indexers are accessed via `.[index]` syntax instead of `[index]` syntax.
```

Comment:
```suggestion
In the previous snippet, the FS0003 error is shown because in F# indexers are accessed via `.[index]` syntax instead of `[index]` syntax.
```